index.html

```diff
@@ -9,13 +9,25 @@
     <link rel="stylesheet" href="styles.css">
   </head>
   <body>
-  
-    <!-- START HEADER -->
+   <header>
 
-    <!-- END HEADER -->
+    <h1>Juan Fabrega</h1> <h3>posts</h3><h3>about</h3>
```

Comment:
@Masonrivers let's try to break each element into new lines so it is easier to read, so instead of this:

`<h1>Juan Fabrega</h1> <h3>posts</h3><h3>about</h3>`

It'd be better like this 
```
<h1>Juan Fabrega</h1>
<h3>posts</h3>
<h3>about</h3>
```

Reply:
Sorry the notifications weren't popping up

Yeah ill space them!